x/concentrated-liquidity/incentives_test.go

```diff
@@ -4,20 +4,60 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	cl "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity"
+	"github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/model"
 	"github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/types"
 )
 
 var defaultPoolId = uint64(1)
 
+type ExpectedUptimes struct {
+	emptyExpectedAccumValues []sdk.DecCoins
+	hundredTokensSingleDenom []sdk.DecCoins
+	hundredTokensMultiDenom []sdk.DecCoins
+	twoHundredTokensMultiDenom []sdk.DecCoins
+	varyingTokensSingleDenom []sdk.DecCoins
+	varyingTokensMultiDenom []sdk.DecCoins
+}
+
+// getExpectedUptimes returns a base set of expected values for testing based on the number
+// of supported uptimes at runtime. This abstraction exists only to ensure backwards-compatibility
+// of incentives-related tests if the supported uptimes are ever changed.
+func getExpectedUptimes() ExpectedUptimes {
```

Comment:
Note to reviewer: want to highlight that this setup is intended to ensure that changing the number of supported uptimes does not break all of our pool-related CL tests